pkg/assets/rte/assets.go

```diff
@@ -10,6 +10,9 @@ var SELinuxInstallSystemdServiceTemplate []byte
 //go:embed selinuxpolicy.cil
 var SELinuxPolicy []byte
 
+//go:embed selinuxpolicy-latest.cil
+var SELinuxPolicyLatest []byte
```

Comment:
`Latest` is a floating tag :) Let's call it `SELinuxPolicyOCP411`

Reply:
Ack, will update now.